### PR TITLE
#426 proposition subpage

### PR DESCRIPTION
**Pull request by estyxx**
This PR 

- Add `external_link` to EmbedPlusCTABlock
- Don't show client-book's title if there aren't any client
- Add embed URL video to Processes panel in service page

### CODE REVIEW

File: tbx/project_styleguide/templates/patterns/organisms/process-block/process-block.html [hunk 1]
```diff
@@ -9,6 +11,12 @@
             </div>
         {% endif %}
 
+        {% if processes_section_embed_url %}
+                <div class="process__embed">
+                    {% embed processes_section_embed_url %}
```
Comment (victoriachan) [inline tbx/project_styleguide/templates/patterns/organisms/process-block/process-block.html:16]: Question: Do we need Frontend dev to look at this to ensure the formatting, size, etc. works across different widths?

Comment (estyxx) [reply tbx/project_styleguide/templates/patterns/organisms/process-block/process-block.html:16]: Probably... actually the style .process__embed doesn't exist and I was about to copy past from `_embed-cta.scss`...

Comment (estyxx) [reply tbx/project_styleguide/templates/patterns/organisms/process-block/process-block.html:16]: yeah I just fixed a bit the styles, took from [there](https://github.com/torchbox/wagtail-torchbox/blob/master/tbx/static_src/sass/components/_streamfield.scss#L30), it's nicer now but would be great to have a FE dev opinion :D

Comment (estyxx) [reply tbx/project_styleguide/templates/patterns/organisms/process-block/process-block.html:16]: CleanShot 2023-03-20 at 12 50 27

Comment (victoriachan) [reply tbx/project_styleguide/templates/patterns/organisms/process-block/process-block.html:16]: Yes, you might want to ask a FE dev. Unfortunately I can't help with this.

**Pull request merged**